gradle-sls-packaging/src/main/groovy/com/palantir/gradle/dist/service/gc/GcProfile.java

```diff
@@ -91,7 +91,7 @@ public final void newRatio(int newerRatio) {
     class Hybrid implements GcProfile {
         @Override
         public final List<String> gcJvmOpts(JavaVersion javaVersion) {
-            return ImmutableList.of("-XX:+UseG1GC");
+            return ImmutableList.of("-XX:+UseG1GC", "-XX:+UseNUMA", "-XX:+PrintFlagsFinal");
```

Comment:
I’m not sure we should make PrintFlagsFinal dependent on the GC profile

Reply:
Fair enough -- I've removed it. 👍